JapaneseFilter/sections/specific.txt

```diff
@@ -60,6 +60,11 @@ gundamlog.com###more > iframe[src^="https://richlink.blogsys.jp"]
 !
 ! NOTE: Specific rules
 !
+reashu.com##section.entry-content .memo:has(> a[id^="h2_ue_ad"] img)
+reashu.com##section.entry-content .say.right:has(> .chatting > .sc > p a[href^="https://px.a8.net/"])
+reashu.com##section.entry-content .sng-box .box-content .center:has(> a[href^="https://px.a8.net/"] img)
+reashu.com##section.entry-content > .sng-box:has(> .box-content > p > a[href^="https://px.a8.net/"] img)
+reashu.com##section.entry-content > .sng-box:has(> ul > li a[href^="https://px.a8.net/"] img)
```

Comment:
Actually, `section` is not needed here `section.entry-content`, but it's also ok.

Reply:
Yeah, removed by https://github.com/AdguardTeam/AdguardFilters/commit/a8ad04e4652834d2d69f0a33ae69db16c1444b5a for simplicity.